app/src/main/java/com/example/cs4520_twitter/data_layer/api/models/AddBabRequest.kt

```diff
@@ -0,0 +1,3 @@
+package com.example.cs4520_twitter.data_layer.api.models
+
+data class AddBabRequest(val content: String)
```

Comment:
This is just a question. Does the api handle the date and default like count or would we have to add this in here?